db.go

```diff
@@ -896,7 +896,7 @@ func (db *DB) Snapshot(dir string, withHead bool) error {
 	if dir == db.dir {
 		return errors.Errorf("cannot snapshot into base directory")
 	}
-	if _, err := ulid.Parse(dir); err == nil {
+	if _, err := ulid.ParseStrict(dir); err == nil {
```

Comment:
can you also do the same change for `isBlockDir`

Reply:
actually you should be able to use `isBlockDir` here as well

Reply:
May not use isBlockDir here, the parameter is different.